utils/_decorators.py

```diff
@@ -106,109 +94,52 @@ def _should_skip(condition=None, library=None, weblog_variant=None):
     return True
 
 
-def missing_feature(condition: bool = None, library=None, weblog_variant=None, reason=None, force_skip: bool = False):
-    """decorator, allow to mark a test function/class as missing"""
-
-    skip = _should_skip(library=library, weblog_variant=weblog_variant, condition=condition)
-
-    def decorator(function_or_class):
-
-        if inspect.isclass(function_or_class):
-            assert condition is not None or (library is None and weblog_variant is None), _MANIFEST_ERROR_MESSAGE
-
-        if not skip:
-            return function_or_class
-
-        full_reason = "missing_feature" if reason is None else f"missing_feature ({reason})"
-
-        return _get_expected_failure_item(function_or_class, full_reason, force_skip=force_skip)
+def decorator(expected_to_fail, force_skip, condition, decorator_type, reason, function_or_class):
```

Comment:
Prevent this method to be exported by error : 

```suggestion
def _decorator(expected_to_fail, force_skip, condition, decorator_type, reason, function_or_class):
```

Reply:
Could you also make `decorator_type`  an string enum, rather a free string ?